doc/devel/color_changes.rst

```diff
@@ -0,0 +1,133 @@
+.. _color_changes:
+
+*********************
+Default Color changes
+*********************
+
+As discussed at length elsewhere [insert links], ``jet`` is an
+empirically bad color map and should not be the default color map.
+Due to the position that changing the appearance of plot is a back
+compatibility breaking change, this change has been put off for far
+longer than it should have been.  In addition to changing the default
+color map we plan to take the chance to change the default color-cycle
+on plots and to adopt a different color map for filled plots (``imshow``,
+``pcolor``, ``contourf``, etc) and for scatter like plots.
+
+
+Default Heat Map Colormap
+-------------------------
+
+The choice of a new color map is fertile ground to bike-shedding ("No,
+it should be _this_ color") so we have a proposed set criteria (via
+Nathaniel Smith) to evaluate proposed color maps.
+
+- it should be a sequential colormap, because diverging colormaps are
+  really misleading unless you know where the "center" of the data is,
+  and for a default colormap we generally won't.
```

Comment:
for me, it ain't just about the 0, but also the data around 0.  I have for simplicity forced the colormap to go between -A and +A where ``A = np.max(np.abs(data))``, but then if the data say goes from -10 to +100, then it comes a bit more complicated so as to avoid the -10 looking quite faint...

This, I think, comes under the realm of the ``Normalize`` class, perhaps we need a few more Normalise classes and think about which one to use as default...